vcl_templates/www.vcl.erb

```diff
@@ -355,6 +384,13 @@ sub vcl_deliver {
     add resp.http.Set-Cookie = "ABTest-EducationNavigation=" req.http.GOVUK-ABTest-EducationNavigation "; expires=" now + 365d "; path=/";
   }
 
+  # Begin dynamic section
+  <% ab_tests.each do |test| %>
+  if (req.http.Cookie !~ "ABTest-<%= test['name'] %>" || req.url ~ "[\?\&]ABTest-<%= test['name'] %>") {
```

Comment:
We should probably add the crawler test to this condition - see line 378. I don't think it will make any practical difference, but it seems wrong add `Set-Cookie` to every response to the crawler.